controllers/home.js

```diff
@@ -29,6 +30,16 @@ async function home(req, res) {
     return res.redirect("/user/dashboard");
   }
 
+  const experimentFlags = getExperimentFlags(req, EXPERIMENTS_ENABLED);
+
+  // Growth Experiment
+  if (EXPERIMENTS_ENABLED) {
+    getExperimentBranch(req, false, ["de", "fr"], {
+      "va": 25,
+      "vb": 25,
+    });
```

Comment:
This uses the same syntax as [Traffic Cop](https://github.com/mozilla/trafficcop/blob/master/documentation.md#type-a-callback)